src/components/autocomplete/autocomplete.spec.ts

```diff
@@ -1,96 +1,90 @@
-          <div class="sbb-gap-fix-wrapper">
-            <div class="sbb-gap-fix-corner" id="right"></div>
-          </div>
-        </div>
-        <div class="sbb-autocomplete__panel">
-          <div class="sbb-autocomplete__wrapper">
-            <div class="sbb-autocomplete__options" ${!isSafari() ? listboxAttr : ''}>
-              <slot></slot>
-            </div>
-          </div>
-        </div>
-      </div>
-    `);
-  });
+  describe('renders standalone', async () => {
+    let elem: SbbAutocompleteElement;
 
-  it('renders in form field', async () => {
-    const root = await fixture(html`
-      <sbb-form-field>
-        <input />
-        <sbb-autocomplete>
+    beforeEach(async () => {
+      await fixture(html`
+        <div id="origin"></div>
+        <input id="trigger" />
+        <sbb-autocomplete origin="origin" trigger="trigger">
           <sbb-option value="1">1</sbb-option>
           <sbb-option value="2">2</sbb-option>
         </sbb-autocomplete>
-      </sbb-form-field>
-    `);
-    const elem = root.querySelector('sbb-autocomplete');
-    const listboxAttr = 'id="sbb-autocomplete-4" role="listbox"';
+      `);
+      elem = document.querySelector('sbb-autocomplete')!;
+      await waitForLitRender(elem);
+    });
 
-    expect(root).dom.to.be.equal(`
-      <sbb-form-field error-space="none" size="m" width="default" data-input-empty data-input-type="input" data-slot-names="unnamed">
-        <input
-          aria-autocomplete="list" aria-controls="sbb-autocomplete-4" aria-expanded="false" aria-haspopup="listbox"
-          aria-owns="sbb-autocomplete-4" autocomplete="off" role="combobox">
-        <sbb-autocomplete data-state="closed" dir="ltr" ${isSafari() ? listboxAttr : ''}>
-          <sbb-option value="1" aria-disabled="false" aria-selected="false" data-variant="autocomplete" id="sbb-option-7" role="option" data-slot-names="unnamed">1</sbb-option>
-          <sbb-option value="2" aria-disabled="false" aria-selected="false" data-variant="autocomplete" id="sbb-option-8" role="option" data-slot-names="unnamed">2</sbb-option>
-        </sbb-autocomplete>
-      </sbb-form-field>
-    `);
-    expect(elem).shadowDom.to.equal(`
-      <div class="sbb-autocomplete__gap-fix"></div>
-      <div class="sbb-autocomplete__container">
-        <div class="sbb-autocomplete__gap-fix">
-          <div class="sbb-gap-fix-wrapper">
-            <div class="sbb-gap-fix-corner" id="left"></div>
-          </div>
-          <div class="sbb-gap-fix-wrapper">
-            <div class="sbb-gap-fix-corner" id="right"></div>
-          </div>
-        </div>
-        <div class="sbb-autocomplete__panel">
-          <div class="sbb-autocomplete__wrapper">
-            <div class="sbb-autocomplete__options" ${!isSafari() ? listboxAttr : ''}>
-              <slot></slot>
-            </div>
-          </div>
-        </div>
-      </div>
-    `);
+    (!isSafari() ? describe : describe.skip)('Chrome-Firefox', async () => {
```

Comment:
```suggestion
    // On top of the file
    const describeSkipSafari = !isSafari() ? describe : describe.skip;
    //...
    describeSkipSafari('Chrome-Firefox', async () => {
```
Do you think something like this could improve readability? Also we would only check if it's Safari once per test run

Reply:
Good idea! 
I would implement something like this in an utility file 
```js
const describeIf = (condition: boolean) => condition ? describe : describe.skip;
const testIf = (condition: boolean) => condition ? test : test.skip;

describeIf(isSafari())('aaaa', () => {});
```

Reply:
Looks a lot better 👍

Reply:
I implemented it a bit differently, have a look if you like it